storage/innobase/trx/trx0rseg.cc

```diff
@@ -34,6 +34,7 @@ Created 3/26/1996 Heikki Tuuri
 
 #ifdef WITH_WSREP
 # include <mysql/service_wsrep.h>
+#include <string>
```

Comment:
Why does this header need to be included? We are not adding any use of `std::string`. The C header `<string.h>` or its C++ wrapper `<cstring>`, which is needed for functions like `memcmp()`, should already be included via other means.

Reply:
Its not needed, I used it when debugging the problem.